### PR TITLE
feature/DPP-93

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: packages/core/src/regex/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const EMAIL_ADDRESS_VALIDATION_REGEX = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```
Comment (sksadjad) [inline packages/core/src/regex/index.ts:1]: add an empty line here

File: packages/core/src/index.ts [hunk 2]
```diff
@@ -3,4 +3,5 @@ import Localization from './localization/Localization'
 export * from './utils'
 export * from './types'
 export * from './styles'
+export * from './regex'
```
Comment (BtencateSphereon) [inline packages/core/src/index.ts:6]: file structure in the project uses plural

File: packages/core/src/regex/index.ts (hunk 1 above)
Comment (BtencateSphereon) [inline packages/core/src/regex/index.ts:1]: export this as camel case

Comment (maikel-maas) [reply packages/core/src/regex/index.ts]: Fixed

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**